main.css~

```diff
@@ -0,0 +1,40 @@
+body {
```

Comment:
It seems your editor may have been creating temporary files and that's why you've committed a few of these ones ending with the `~` character. If they're not useful to the submission you should try and remove them from the project.